server/build/babel/find-config-location.js

```diff
@@ -0,0 +1,16 @@
+import { join } from 'path'
+import buildConfigChain from 'babel-core/lib/transformation/file/options/build-config-chain'
```

Comment:
Otherwise, we have to copy the code or logic.
I thought this method might better.